src/ci/docker/host-x86_64/x86_64-gnu-tools/Dockerfile

```diff
@@ -67,12 +67,14 @@ COPY host-x86_64/x86_64-gnu-tools/checktools.sh /tmp/
 RUN curl -sL https://nodejs.org/dist/v14.4.0/node-v14.4.0-linux-x64.tar.xz | tar -xJ
 ENV PATH="/node-v14.4.0-linux-x64/bin:${PATH}"
 
+COPY scripts/browser-ui-test.version /tmp/
+
 # For now, we need to use `--unsafe-perm=true` to go around an issue when npm tries
 # to create a new folder. For reference:
 # https://github.com/puppeteer/puppeteer/issues/375
 #
 # We also specify the version in case we need to update it to go around cache limitations.
-RUN npm install -g browser-ui-test@0.8.1 --unsafe-perm=true
+RUN sh npm install -g browser-ui-test@`cat /browser-ui-test.version` --unsafe-perm=true
```

Comment:
Please also leave a comment here that the file is consumed by bootstrap as well.

Reply:
Ok!